pyrasterframes/src/main/python/pyrasterframes/rf_ipython.py

```diff
@@ -31,7 +31,7 @@ def tile_to_png(tile, fig_size=None):
     from matplotlib.figure import Figure
 
     # Set up matplotlib objects
-    nominal_size = 4  # approx full size for a 256x256 tile
+    nominal_size = 2  # approx full size for a 256x256 tile
```

Comment:
makes sense now to remove that code comment

These seemed small in the docs to me. but it is subjective.

![image](https://user-images.githubusercontent.com/7798319/63115068-62660a00-bf64-11e9-9ba6-cf9c41c45c3d.png)

another example

![image](https://user-images.githubusercontent.com/7798319/63115111-790c6100-bf64-11e9-9cae-cbfe46181bf8.png)

Reply:
Bumped to 3

Reply:
Bumped to 3